### PR TITLE
Link to the license and minor changes

### PR DESCRIPTION
**Pull request by shavidissa**

### CODE REVIEW

**Comment by shavidissa** [comment]
**Under review: do not merge!**

CC @rkempf42

File: README.md [hunk 1]
```diff
@@ -4,12 +4,12 @@
...
+* [Prerequisites](#Prerequisites)
 * [Auto Configuration](#auto-configuration)
 * [Custom Configuration](#custom-configuration)
 * [FAQ](#faq)
 
-## Prerequisite
+## Prerequisites
 
 * Java 8 or above.
 * Introduce the latest `wavefront-spring-boot-starter` to your application by adding the following dependency to your pom.xml file.
```
Comment (rkempf42) [inline README.md]: How about: 
Add the following ... to introduce the latest...

File: README.md [hunk 2]
```diff
@@ -23,7 +23,7 @@
 
 ## Auto Configuration
 
-Once the application starts, you get a single-use login URL. Use it to log in to the Wavefront service and access the data collected via the browser.
+Once the application starts, you get a single-use-login URL. Use it to log in to the Wavefront service and access the data collected via the browser.
```
Comment (rkempf42) [inline README.md]: Once the application starts, you get a single-use-login URL. Use it to log in to the Wavefront service and access the data collected via the browser.
=>
When the application starts, you get a single-use login URL. Use it to log in to the Wavefront service and access the data that wavefront-springboot-starter collected.

File: README.md [hunk 3]
```diff
@@ -23,7 +23,7 @@
 
 ## Auto Configuration
```
Comment (rkempf42) [inline README.md]: Do we want a heading that says Auto Configuration?

Comment (shavidissa) [reply README.md]: As discussed, leaving this heading for now because it makes sense with the other heading (Custom Configuration).

**Comment by shavidissa** [comment]
Thank you! @rkempf42

**Comment by shavidissa** [comment]
CC @panghy @sushantdewan123 : Can you merge the following changes, please?

**Pull request merged**